fil-proofs-tooling/src/bin/benchy/prodbench.rs

```diff
@@ -254,15 +254,11 @@ pub fn run(
 struct CircuitOutputs {
     // porep_snark_partition_constraints
     pub porep_constraints: usize,
-    // replica_inclusion (constraints: single merkle path pedersen)
     // data_inclusion (constraints: sha merklepath)
     // window_inclusion (constraints: merkle inclusion path in comm_c)
     // ticket_constraints - (skip)
-    // replica_inclusion (constraints: single merkle path pedersen)
-    // column_leaf_hash_constraints - (64 byte * stacked layers) pedersen_md
     // kdf_constraints
     // merkle_tree_datahash_constraints - sha2 constraints 64
-    // merkle_tree_hash_constraints - 64 byte pedersen
 }
```

Comment:
Shouldn't these all be replaced with poseidon, rather than simply removed?

Reply:
seems like these comments are not really used anymore anyway

Reply:
Maybe kill them all then?

Reply:
done